PC/hu_moments_gather/hu_moments_gather.cpp

```diff
@@ -92,6 +94,37 @@ int main(int argc, char** argv)
     return 0;
 }
 
+vector<Point> largestContourConvexHull(const vector<vector<Point> >& contours )
+{
+    vector<Point> result, pts;
+    std::pair<double, vector<Point>> maxArea;
+
+    for ( size_t i = 0; i < contours.size(); i++)
+    {
+        vector<Point> tmpPts;
+        for ( size_t j = 0; j< contours[i].size(); j++)
+        {
+            tmpPts.push_back(contours[i][j]);
+            pts.push_back(contours[i][j]);
+        }
+
+        auto contourAreaSize = contourArea(tmpPts);
+//        auto contourLength = arcLength(tmpPts, false); //false indicates it is not closed curve
+//        std::cout << "Area: " << contourAreaSize << '\n';
+//        std::cout << "Length " << contourLength << '\n';
```

Comment:
Do we want to keep some debug messages? We can introduce a function or macro to handle this. Otherwise, remove commented out code.

Reply:
It s no longer necessary. Will be removed.